usaspending_api/database_scripts/matview_generator/mv_contract_award_search.json

```diff
@@ -22,11 +22,12 @@
     "  awards.piid,",
     "  NULL::text AS fain,",
     "  NULL::text AS uri,",
-    "  awards.total_obligation,",
+    "  COALESCE(awards.total_obligation, 0)::NUMERIC(23, 2) AS award_amount,",
+    "  COALESCE(awards.total_obligation, 0)::NUMERIC(23, 2) AS total_obligation,",
```

Comment:
Do we still need the alias `total_obligation` or is it being left to not break current functionality?

Reply:
`total_obligation` is the value returned by the API